deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml

```
@@ -71,6 +71,11 @@ spec:
                 - force
                 - version
                 type: object
+              proceed:
+                default: true
+                description: Procceed whether to upgrade or not, this gives operator
+                  the ability to cancel the upgrade if upgrade hasn't started
```

Just very minor spelling/grammar fix
```suggestion
                description: Proceed whether to upgrade or not, this gives the operator
                  the ability to cancel the upgrade if the upgrade hasn't started
```